### PR TITLE
Buzzer improvement and first draft of new reporting

### PR DESCRIPTION
**Pull request by ihsgnef**
Buzzer improvement:
- batch normalization, training & valid accuracy is higher but final performance is worse (with old reporting code). I'll do a hyperparameter search once I finish the new reporting code
- Add average pretrained word2vec to the input vector of buzzer (pretrained word2vec file goes to data/external?)

Buzzer-Expo-Reporting:
- Fixed a bug: buzzes are not masked when output. This bug doesn't affect downstream application, it's just generating buzzes after the end of the sentence
- Buzzer now provides more information to the expo (not displayed yet)
- New reporting: under construction

### CODE REVIEW

**Comment by EntilZha** [comment]
Is there something specific you wanted me to look at. LGTM code-wise. Is there a sample report/plot I could look at?
